src/Uno.UWPSyncGenerator/Generator.cs

```diff
@@ -136,36 +136,45 @@ abstract class Generator
 		static Generator()
 		{
 			RegisterAssemblyLoader();
-		}
-
-		public virtual void Build(string basePath, string baseName, string sourceAssembly)
-		{
 			Directory.SetCurrentDirectory(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location));
 			InitializeRoslyn();
```

Comment:
I'm assuming `InitializeRoslyn` is needed to be called only once. So I moved to the static ctor. Let me know if that's not correct.

Reply:
That's fine!